docs/concepts/unions.md

```diff
@@ -9,8 +9,8 @@ Validating unions feels like adding another orthogonal dimension to the validati
 
 To solve these problems, Pydantic supports three fundamental approaches to validating unions:
 
-1. [left to right mode](#left-to-right-mode) - the simplest approach, each member of the union is tried in order
-2. [smart mode](#smart-mode) - as with "left to right mode" all members are tried, but strict validation is used to try to find the best match, this is the default mode for most union validation
+1. [left to right mode](#left-to-right-mode) - the simplest approach, each member of the union is tried in order and the first match is returned
+2. [smart mode](#smart-mode) - similar to "left to right mode" members are tried in order, and validation will proceed past the first match to attempt to find a better match, this is the default mode for most union validation
```

Comment:
```suggestion
2. [smart mode](#smart-mode) - similar to "left to right mode" members are tried in order, however validation will proceed past the first match to attempt to find a better match, this is the default mode for most union validation
```

Reply:
I think you need a `;` instead of a comma in that case